packages/content/src/lib/content-files-list-token.ts

```diff
@@ -3,6 +3,11 @@ import { InjectionToken } from '@angular/core';
 import { ContentFile } from './content-file';
 import { getContentFilesList } from './get-content-files';
 
+function getSlug(filename: string) {
+  const parts = filename.match(/^(\\|\/)(.+(\\|\/))*(.+)\.(.+)$/);
```

Comment:
Will this support nested files?

Such as

/src/content/posts/my-post.md

Some tests for the regex would be nice also

Reply:
Yes it will. Also Windows / Unix Path names. But I agree: tests would be neat.